File: android_sqlite/app/src/main/java/top/zcwfeng/sqlite/MainActivity.java
package top.zcwfeng.sqlite;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import top.zcwfeng.sqlite.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding = null;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: android_hilt/koinsample/src/main/java/top/zcwfeng/koinsample/MainActivity.kt
package top.zcwfeng.koinsample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import org.koin.android.ext.android.inject
import top.zcwfeng.koinsample.data.User
import top.zcwfeng.koinsample.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private val user: User by inject()
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
//        binding.userTextView.text = "${user.name}使用 Koin 的心情是：${user.mood}"
        user.mood = "心情复杂"

    }
}
